heartbeat/tests/system/test_monitor.py

```diff
@@ -74,7 +74,7 @@ def test_http_json(self, expected_status, body):
             self.render_config_template(
                 monitors=[{
                     "type": "http",
-                    "urls": ["http://localhost:{}".format(server.server_port)],
+                    "hosts": ["http://localhost:{}".format(server.server_port)],
```

Comment:
We need a variant of this test that uses "urls". I'd actually be fine with it just being basically a copy/paste of this test (or, if you'd like just do it dynamically with to generate two different test cases)